packages/core/package.json

```diff
@@ -39,17 +38,12 @@
     "@truffle/migrate": "^3.2.31",
     "@truffle/plugins": "^0.2.3",
     "@truffle/preserve": "^0.2.3",
-    "@truffle/preserve-fs": "^0.2.3",
-    "@truffle/preserve-to-buckets": "^0.2.3",
-    "@truffle/preserve-to-filecoin": "^0.2.3",
-    "@truffle/preserve-to-ipfs": "^0.2.3",
```

Comment:
These need to stick around because they are included as [`defaultPlugins`](https://github.com/trufflesuite/truffle/blob/develop/packages/core/lib/commands/preserve/plugins.js#L3-L8)

Reply:
Reverted. I think we should have integration tests for these.